src/snowflake/cli/api/project/schemas/native_app/native_app.py

```diff
@@ -32,6 +32,10 @@ class NativeApp(UpdatableModel):
         title="Identifier of the stage that stores the application artifacts.",
         default="app_src.stage",
     )
+    scratch_stage: Optional[str] = Field(
+        title="Identifier of the stage that stores temporary scratch data used by the Snowflake CLI.",
+        default="app_src.stage_snowflake_cli_scratch",
```

Comment:
I got an earlier comment about making this a template, but template rendering actually happens on the parsed `snowflake.yml` before resolving this `default` value, so we can't use templates here.

Reply:
@sfc-gh-melnacouzi we have multiple defaults we could now phrase like templates -- source stage, application name, and package name. What's the right way for us to proceed here?

@sfc-gh-fcampbell as the originator of the comment you're talking about, this is not a blocker for this PR.

Reply:
Agreed, let's go with this for now, we can introduce the templates later on.